Properties/AssemblyInfo.cs

```diff
@@ -6,11 +6,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("GitTfs.GitExtensions.Plugin")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyDescription("A GitExtensions plugin for git-tfs.")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyCompany("Nathan Roe")]

```

Comment:
I thought that Nathan Roe (@roend83) was the original author of this plugin (see commit/issues history). I'm not sure about licensing and stuff, but it can be changed to 'Git-Tfs team' if that's the case.